scanners/pshtt.py

```diff
@@ -1,82 +1,85 @@
 import logging
-from scanners import utils
-import os
-import json
-import glob
 
-###
-# == pshtt ==
-#
-# Inspect a site's TLS configuration using DHS NCATS' pshtt tool.
-#
-###
+import re
+from pshtt import pshtt
 
-command = os.environ.get("PSHTT_PATH", "pshtt")
```

Comment:
In addition to porting the pshtt scanner to work on Lambda, I also ported it to use the Python API rather than shelling out to `pshtt`. We don't reference a `command` or environment variables for anything.